bin/run-command-on-git-revisions

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
I don't know 'cause I didn't write it or try it on say fsh or csh zsh or tcsh.   I have some more unpushed code for this that tests the cache better... Maybe I could strip this down.  Main thing is running commands that loaf the stack safely across revisions.

B mobile phone

> On Dec 27, 2015, at 4:14 PM, Lucas Hosseini <notifications@github.com> wrote:
> 
> In bin/run-command-on-git-revisions:
> 
> > @@ -0,0 +1,86 @@
> > +#!/bin/bash
> I don't feel strongly about this, but do we really need bash or would any POSIX-compliant shell work here?
> 
> —
> Reply to this email directly or view it on GitHub.
>